libcontainer/seccomp/seccomp_unsupported.go

```diff
@@ -0,0 +1,3 @@
+// +build windows
```

Comment:
should this be `// +build !linux` to make it more clear (e.g. linux vs non-linux)